spec/factories/application.rb

```diff
@@ -31,13 +31,13 @@
         "student0_application_code_samples": FFaker::Lorem.paragraph,
         "student0_application_learning_history": FFaker::Lorem.paragraph,
         "student0_application_skills": FFaker::Lorem.paragraph,
-        "student0_application_code_background": FFaker::Lorem.paragraph,
+        "student0_application_language_learning_period": FFaker::Lorem.paragraph,
```

Comment:
For some reason adding `User::MONTHS_LEARNING.sample` gives me a ton of failures on the test suite (`mentor/applications_controller` and `mentor/application` specs), all related to the `JSON::ParserError` I was getting before with my local application. Because of this I'm wondering whether that parsing error is an error with my local data after all – the test data shown when running the tests gives me stuff like `"student1_application_language_learning_period": 10-12` which as you and Carsten already pointed out, is invalid. 
I'm confused and not sure how to best look into this 😭